demo

```diff
@@ -141,8 +141,8 @@ trap exitfunc SIGTERM
 echo "Starting the Mender demo environment..."
 
 docker-compose \
-    -f docker-compose.yml \
     -f docker-compose.storage.minio.yml \
+    -f docker-compose.yml \
```

Comment:
It is okey, but I think that the order was not a problem. What fixes the issue is your later fix (missing storage.minio the docker-compose files in the logs call).